servlet/pom.xml

```diff
@@ -40,6 +40,11 @@
       <artifactId>togglz-test-harness</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
Do be honest, I would like to avoid additional dependencies here. As you see `togglz-servlet` only depends on `togglz-core` and the Servlet API. I think you don't use much from this dependency and it should be easy to do it without these classes.